packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -488,3 +486,51 @@ export class LoggingError extends Error {
         return props;
     }
 }
+
+export class DataCorruptionError extends LoggingError implements IErrorBase {
```

Comment:
i don't really care about error base. i don't think it's very useful. you can then move the dataCorruptioError contant here, and then no need to referenence container-utils